### DISCUSSION TITLE
Is there a better storage method in Qrant cluster mode besides using network storage volumes

### DISCUSSION THREAD

**Discussion by yesletgo** [qdrant/qdrant#2495] [General]
In the qrant cluster mode, can only network mounted disks be used for storage? Currently, Network mounted disks are used, but there are often some errors. For example,DeadlineExceeded messages, such as "Healthcheck timeout 2000ms exceeded"

the IOPS of those network mounted disks :    
IOPS=62.0k, BW=246MiB/s (258MB/s)(8192MiB/33299msec).

**Comment by timvisee** [comment]
This totally depends on the machines (nodes) you use to build up your cluster.

If you're using Kubernetes, then yes, you're likely using network mounted disks. If you use bare metal machines with local NVMe disks, performance would look a lot different.

All nodes obviously talk to each other over the network.

A "Healthcheck timeout 2000ms exceeded" message has to do with a timeout between nodes, not with disks.

**Reply by yesletgo** [reply]
I am using Kubernetes. What can I do to reduce this situation?

**Reply by yesletgo** [reply]
@timvisee  Can the method you mentioned be used for the qdrant cluster installed by Kubernetes？What I understand is that the cluster installed in this way can only use network disks for shared storage to storage the data of qdrant.

**Reply by generall** [reply]
Hi @yesletgo as I understand, that's up to you to configure k8s in a way you want to it store files. Qdrant have no control over it
